pkgs/development/r-modules/wrapper-rstudio-server.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, runCommand, R, fontconfig, rstudio-server, wrapQtAppsHook, recommendedPackages, packages, qtbase }:
+
+let
+  qtVersion = with lib.versions; "${major qtbase.version}.${minor qtbase.version}";
+in
+runCommand (rstudio-server.name + "-wrapper") {
```

Comment:
although this is related to R, does it belong in r-modules? It's not really a package containing r code

Reply:
I put it here to be consistent with the rstudio wrapper, I think this is a reasonable place for it.